runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostIndicator.java

```diff
@@ -76,12 +76,12 @@ public Color getTextColor()
 		int boosted = client.getBoostedSkillLevel(skill),
 			base = client.getRealSkillLevel(skill);
 
-		if (boosted > base)
+		if (boosted < base)
```

Comment:
This should be boosted <= base

Reply:
I guess it does not really matter, the color is not applied when not boosted anyway